.env

```diff
@@ -0,0 +1 @@
+GOOGLE_API_KEY=AIzaSyBOuToubYd5Vqe5Mcp78zfGRVEh7YVgN-g
```

Comment:
doh! add `.env` to your gitignore or this defeats the purpose of putting this in the gitignore.